### PR TITLE
user/ddclient: new package (3.11.2)

### PR DESCRIPTION
**Pull request by Gnarwhal**

### CODE REVIEW

File: user/ddclient/files/ddclient [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# ddclient service
+
+type       = process
+command    = /usr/bin/ddclient -foreground
+depends-on = network.target
+restart    = true
+logfile    = /var/log/ddclient.log
```
Comment (nekopsykose) [inline user/ddclient/files/ddclient:7]:
```suggestion
type = process
command = /usr/bin/ddclient -foreground
depends-on = network.target
logfile = /var/log/ddclient.log
```

File: user/ddclient/template.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+]
+depends = [
+    "curl",
+    "perl",
+]
+pkgdesc = "Client used to update dynamic DNS entries"
+maintainer = "Gnarwhal <git.aspect893@passmail.net>"
+license = "GPL-2.0-or-later"
+url = "https://ddclient.net"
+source = "https://github.com/ddclient/ddclient/archive/refs/tags/v3.11.2.tar.gz"
```
Comment (nekopsykose) [inline user/ddclient/template.py:22]:
```suggestion
source = f"https://github.com/ddclient/ddclient/archive/refs/tags/v{pkgver}.tar.gz"
```

File: user/ddclient/template.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+build_style = "gnu_configure"
+make_cmd = "gmake"
+make_dir = "."
+hostmakedepends = [
+    "automake",
+    "gmake",
+]
+makedepends = [
+    "curl",
+]
```
Comment (triallax) [inline user/ddclient/template.py:13]: looks like a mistake
```suggestion
```

Comment (nekopsykose) [reply user/ddclient/template.py:13]: it should be in hostmakedepends

File: user/ddclient/template.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
+pkgname = "ddclient"
+pkgver = "3.11.2"
+pkgrel = 0
+build_style = "gnu_configure"
+make_cmd = "gmake"
+make_dir = "."
```
Comment (nekopsykose) [inline user/ddclient/template.py:6]:
```suggestion
```

these are not needed

**Comment by nekopsykose** [comment]
should be in contrib/

File: user/ddclient/files/ddclient (hunk 1 above)
Comment (Gnarwhal) [reply user/ddclient/files/ddclient:7]: Why shouldn't restart be true?

File: user/ddclient/template.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+pkgdesc = "Client used to update dynamic DNS entries"
+maintainer = "Gnarwhal <git.aspect893@passmail.net>"
+license = "GPL-2.0-or-later"
+url = "https://ddclient.net"
+source = "https://github.com/ddclient/ddclient/archive/refs/tags/v3.11.2.tar.gz"
+sha256 = "243cd832abd3cdd2b49903e1b5ed7f450e2d9c4c0eaf8ce4fe692c244d3afd77"
+
+
+def post_install(self):
+    self.install_license("COPYING")
```
Comment (triallax) [inline user/ddclient/template.py:27]: don't need to install license for gpl
```suggestion
```

File: user/ddclient/files/ddclient (hunk 1 above)
Comment (nekopsykose) [reply user/ddclient/files/ddclient:7]: it is the default

File: user/ddclient/template.py [hunk 6]
```diff
@@ -0,0 +1,25 @@
+pkgname = "ddclient"
+pkgver = "3.11.2"
+pkgrel = 0
+build_style = "gnu_configure"
+hostmakedepends = [
+    "automake",
+    "curl",
+    "gmake",
```
Comment (nekopsykose) [inline user/ddclient/template.py:8]:
```suggestion
```

since it's not used anymore

**Pull request merged**